runtime/doc/lua.txt

```diff
@@ -2203,6 +2203,27 @@ add({filetypes})                                          *vim.filetype.add()*
       • {filetypes}  (table) A table containing new filetype maps (see
                      example).
 
+get_option({filetype}, {option})                   *vim.filetype.get_option()*
```

Comment:
Related vim todo for reference: https://github.com/vim/vim/blob/e764d1b4219e6615a04df1c3a6a5c0210a0a7dac/runtime/doc/todo.txt#L1036-L1038

```
Add options_default() / options_restore() to set several options to Vim
defaults for a plugin. Comments from Zyx, 2017 May 10.
Perhaps use a vimcontext / endvimcontext command block.
```